Live-Snap/Live-Snap/Utility/GenerateLiveWallpaperWithBarcode.swift

```diff
@@ -60,16 +60,10 @@ class GenerateLiveWallpaperWithBarcode {
     
     func interpolateFrames() -> [UIImage]? {
         
-        let frame1 = wallpaperImage
+        guard let frame2Background = wallpaperImage.darkenedAndBlurred(darkness: 0.06, blurRadius: 60) else { return nil }
```

Comment:
I don't think we want the blur radius to be 60 here do we?

Reply:
We can play around with the numbers. 30 maybe?